Project.toml

```diff
@@ -4,6 +4,7 @@ authors = ["CLIMA contributors <clima-software@caltech.edu>"]
 version = "0.1.1"
 
 [deps]
+CVXOPT = "1a0d837d-f7ba-5e72-a8f8-b8273492ab4e"
```

Comment:
Need to add Convex to the depdencies

Reply:
I just updated the dependencies and locally ran through the tests.